CONTRIBUTING.md

```diff
@@ -117,22 +69,70 @@ repo root, run:
 ```sh
 mkdir build
 cd build
-cmake -Duse_default_uuid=ON ..
+cmake ..
 cmake --build .
 ```
+
+#### CMake build options
+The following CMake options are available for adding/removing project features.
+
+<table>
+<tr>
+<td>Option</td>
+<td>Description</td>
+<td>Default Value</td>
+</tr>
+<tr>
+<td>BUILD_TESTING</td>
+<td>Generates Unit Test for compilation. CMake will automatically download and build g-test.<br>After Compiling, use `ctest` to run Unit Test.</td>
+<td>OFF</td>
+</tr>
+<tr>
+<td>BUILD_STORAGE_SAMPLES</td>
+<td>Build Azure Storage clients sample application.</td>
+<td>OFF</td>
+</tr>
+<tr>
+<td>RUN_LONG_UNIT_TESTS</td>
+<td>Enables the special unit tests which takes more than 3 minutes to run. THis tests are for some specific features like the connection pool for curl transport adapter.</td>
+<td>OFF</td>
+</tr>
+<tr>
+<td>WARNINGS_AS_ERRORS</td>
+<td>Warnings will make compiling fail</td>
+<td>ON</td>
+</tr>
+<tr>
+<td>BUILD_CURL_TRANSPORT</td>
+<td>Build the curl http transport adapter. When building on Posix systems, if no other transport adapter is built, this option will be automatically turned ON</td>
+<td>OFF</td>
+</tr>
+<tr>
+<td>BUILD_DOCUMENTATION</td>
+<td>Build Doxygen documentation</td>
+<td>OFF</td>
+</tr>
+</table>
+<br>
+<br>
```

Comment:
Get rid of the extra space

Reply:
I wanted 2 spaces, not only 1.
I think it looks better since it is a table above

Reply:
This extra whitespace on the page (which we don't have anywhere else in all our markdowns), looks better? Hmm. OK.

![image](https://user-images.githubusercontent.com/6527137/95640278-e0328200-0a50-11eb-8c25-f7093353879e.png)

FYI, we should try to limit use of custom HTML tags (like `<br>`) since markdown linters don't like them, and we need to add them as an exception.

Reply:
removed